profile-widget/profilewidget2.cpp

```diff
@@ -644,8 +644,20 @@ void ProfileWidget2::plotDive(struct dive *d, bool force)
 
 #ifndef SUBSURFACE_MOBILE
 	if (plotInfo.maxhr) {
-		heartBeatAxis->setMinimum(plotInfo.minhr);
-		heartBeatAxis->setMaximum(plotInfo.maxhr);
+		int heartBeatAxisMin = lrint(plotInfo.minhr / 5.0 - 0.5) * 5;
+		int heartBeatAxisMax, heartBeatAxisTick;
+		if ((plotInfo.maxhr - plotInfo.minhr) < 40)
```

Comment:
the extra parenthesis are not need here:
`if ((plotInfo.maxhr - plotInfo.minhr) < 40)`
could be:
`if (plotInfo.maxhr - plotInfo.minhr < 40)`

change this and the ones bellow if you want to.
not a big issue.